package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "soap-component-oih",
+  "version": "1.0.0",
+  "description": "This is the soap api component for OIH",
+  "main": "node_modules/@openintegrationhub/ferryman/runGlobal.js",
+  "scripts": {
+    "compile": "eslint . --ext .ts && tsc",
+    "test": "LOG_LEVEL=fatal ts-mocha test --recursive --timeout 50000",
+    "test:debug": "LOG_LEVEL=trace DEBUG=nock.* ts-mocha test --recursive --timeout 50000"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/blendededge/soap-component-oih.git"
+  },
+  "keywords": [
+    "SOAP",
+    "OIH",
+    "Connector"
+  ],
+  "author": "Blended Edge",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/blendededge/soap-component-oih/issues"
+  },
+  "homepage": "https://github.com/blendededge/soap-component-oih#readme",
+  "dependencies": {
+    "@openintegrationhub/ferryman": "^2.1.0",
+    "axios": "^0.25.0",
```

Comment:
Is there a reason to include Axios?

Reply:
Nope, was taking dependencies from the REST component, forgot it wouldn't be necessary